mp4parse/src/boxes.rs

```diff
@@ -55,4 +55,5 @@ box_database!(
     ProtectedVisualSampleEntry 0x656e6376, // "encv" - Need to check official name in spec.
     ProtectedAudioSampleEntry  0x656e6361, // "enca" - Need to check official name in spec.
     MovieExtendsBox            0x6d766578, // "mvex"
+    UserDataBox                0x75647461, // "udta"
```

Comment:
This would be better as a separate commit. It seems unrelated to the rest of the patch.